CONTRIBUTING.md

```diff
@@ -15,3 +15,6 @@
 * Use the same coding style as the rest of the [codebase](https://github.com/jashkenas/underscore/blob/master/modules/index.js).
 
 * In your pull request, do not add documentation or re-build the minified `underscore-umd-min.js` file. We'll do those things before cutting a new release.
+
+## "Help, cloning fails with `fatal: fsck error in packed object`"
```

Comment:
Maybe make this `###` to make clear that it is a subsection of the top-level heading.

(I wondered for a moment whether we should make the top-level heading `#` instead, but I think that is meant to rank below the overall README.)